src/System.IO.FileSystem/tests/FileStream/CopyToAsync.cs

```diff
@@ -47,6 +42,53 @@ public void InvalidArgs_Throws(bool useAsync)
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "This fails on netcoreapp because the Stream CopyToAsync calls Length which checks the validity of the underlying handle. On NetFX the operation no-ops for no input or delays failure to execution for input. See /dotnet/coreclr/pull/4540.")]
```

Comment:
Nit: the comment is a bit confusing as it starts saying "this fails on netcoreapp" and it is being disabled for netfx.